packages/components/src/styles/components/app-side-nav/content.scss

```diff
@@ -130,6 +131,7 @@
   // Important: This element does not doing anything when interacted with so should not change color according to state
   &.active,
   &.active:hover:focus {
+    position: relative; // for the "active" indicator bar to be correctly positioned
```

Comment:
So far, I haven't been able to uncover issues related to this change and the z-index above.

Reply:
Nice!